doc/Quick-Start-Guide.rst

```diff
@@ -114,10 +114,11 @@ Unit Benchmark
 
 Troubleshooting
 ================
-- If the pip installation fails while installing build dependencies,
+- If an installation failure due to ``DistributionNotFound: ipaddress``
+  while installing build dependencies,
   run the following commands::
 
-    sudo python -m pip uninstall pip setuptools
+    sudo pip install ipaddress
```

Comment:
This is a troubleshooting guide. It is meant to describe troubleshooting steps. The least we can do is make this description generic.
Please update the change to something like: "If pip fails to install a package while installing build dependencies, try to manually install the package using: sudo pip install <package name>"
Also, please update install-build-deps script to check for and install 'ipaddress' package.

Reply:
Done.